indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -74,6 +74,26 @@
             $reviewBox.find('.js-edit-review').trigger('click');
             $('body, html').animate({scrollTop: $reviewBox.offset().top}, 'fast');
             $reviewBox.find('.i-timeline-item-box').effect('highlight', {color: Palette.highlight}, 'slow');
+        }).on('click', '.js-toggle-details', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+
+            function toggleDetails($trigger) {
```

Comment:
This functionality is so generic that this could easily go into declarative.js (bind to `[data-toggle]:not([data-toggle=dropdown])`)!